recipes/libelfin/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibelfinConan(ConanFile):
+    name = "libelfin"
+    description = "C++11 library for reading ELF binaries and DWARFv4 debug information"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/aclements/libelfin"
+    license = "MIT"
+    topics = ("conan", "elf", "dwarf", "libelfin")
+
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    exports_sources = "CMakeLists.txt", "patches/*"
+
+    _source_subfolder = "source_subfolder"
+    _cmake = None
+
```

Comment:
```suggestion
    _cmake = None
    
    @property
    def _source_subfolder(self):
        return "source_subfolder"

```

Reply:
I would use `@property` if there was any _behavior_. For example, if value depends on options or settings. For a plain value the original code is all it needs.

Reply:
I understand your point of view and you are right.
However, for the sake of consistency, usually the cached cmake is first... it's harmless to apply the suggestion if you dont mind 🙏

Reply:
Done!